### DISCUSSION TITLE
Adjusting README for profile

### DISCUSSION THREAD

**Discussion by MopyKing** [community/community#22022] [Projects and Issues]
<p>hey guys so, i looked up on google and saw a cool way to show my profile<br>
with gifs and emojis - by writing into the README file in the repo named like my username<br>
so, i put a link to my linked in profile as a hyperlink on this profile<br>
as following :</p>
 <div align="center">
  <a>
  <img src="https://img.shields.io/badge/LinkedIn-blue?style=for-the-badge&amp;logo=linkedin&amp;logoColor=white" alt="LinkedIn Badge" width="111" height="28">
  </a>
<p>the href holds the correct URL<br>
though, when i click on the link in my profile it redirects me into<br>
<a href="https://github.com/MopyKing/MopyKing/blob/d8cfe27c55b7b8dc7f98dec6d8ad2ae4a8adc5ce/www.linkedin.com/in/roy-edri" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/MopyKing/MopyKing/blob/d8cfe27c55b7b8dc7f98dec6d8ad2ae4a8adc5ce/www.linkedin.com/in/roy-edri</a><br>
why is it doing that and how can i fix that ?<br>
thnx</p>
</div>

**Comment by jfullstackdev** [comment]
<p>it’s 404</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/1/61e7a9073cf1fcdd4ea58037a615eed87ba8cc40.jpeg" data-download-href="/uploads/short-url/dY6DXkgHF0aYc2rTi0fWjlwNkhW.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181068304-4210f64c-a2ec-48f2-be6c-391eea156774.jpeg" alt="image" data-base62-sha1="dY6DXkgHF0aYc2rTi0fWjlwNkhW" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61e7a9073cf1fcdd4ea58037a615eed87ba8cc40_2_690x373.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61e7a9073cf1fcdd4ea58037a615eed87ba8cc40_2_1035x559.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61e7a9073cf1fcdd4ea58037a615eed87ba8cc40_2_1380x746.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61e7a9073cf1fcdd4ea58037a615eed87ba8cc40_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 233 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>there are services using URLs, and if you click on that, it will go to the page of the service, maybe that’s the behavior you are referring to</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The link in your README.md file is lacking the protocol part (<code>https://</code>), so GitHub assumes you mean to link to a file in your repository. Fix the link so it contains the full URL. :slightly_smiling_face:</p>

**Comment by MopyKing** [comment]
<p>haha omg, yea that was the problem thanks :smiley:</p>
